### DISCUSSION TITLE
[Projects Beta] Subtasks/issues/dependencies

### DISCUSSION THREAD

**Discussion by simon-ourmachinery** [community/community#9913] [Projects and Issues] [answered]
It would be cool if I could group issues together and sub hierarchy such as I can do in workflowy

1. Issue Number 1
1.1 Issue Number 2 needs to be finished first
1.2 Issue Number 3 needs to be finished next
1.2 Issue Number 4 needs to be done
2. Issue number 4

**Comment by zonofthor** [comment]
Same as requested here
 https://github.com/github/feedback/discussions/5714

and
 https://github.com/github/feedback/discussions/7813

i.e. this is a highly requested feature!

**Comment by white-sdev** [comment]
Please! make them with infinite sublevels, It is very anoying to find yourself limited by just 1 level, this will be deal braker for me :(
1- First Task
  1.1 SubT Level 2
    1.1.1 SubT1 Level 3
      1.1.1.1 subtT1 Level 4
  1.2 Level2
    1.2.1 subt1 Level 3
    1.2.2 subt2 Level 3
    1.2.3 subt3 Level 3
  1.3 Level 2
2- Level 1

**Comment by stilliard** [comment]
Dependencies or blocking issue relations would be very helpful.

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋 Tasklists add support for issue hierarchy at GitHub and are currently in Private Beta! You can request access and join the waitlist [here](https://github.com/features/issues/signup).

For more information on how these work, check out the [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists).

**Reply by esmarkowski** [reply]
I want this so so much
